From 1177f87da0eadabf76736966d1a30e9a2e5b416a Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:17:29 +0200
Subject: [PATCH 1/2] add a subtraction operation

Add member-function which performs a subtraction in terms of
an addition.

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
---
 calculator.cpp | 5 +++++
 calculator.h   | 1 +
 2 files changed, 6 insertions(+)

diff --git a/calculator.cpp b/calculator.cpp
index 23b2bb3..3c63184 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -4,3 +4,8 @@ int Calculator::Add (int a, int b)
 {
     return a + b;
 }
+
+int Calculator::Sub (int a, int b)
+{
+    return Add (a, -b);
+}
diff --git a/calculator.h b/calculator.h
index 6b87fdc..5647eab 100644
--- a/calculator.h
+++ b/calculator.h
@@ -2,4 +2,5 @@ class Calculator
 {
     public:
         int Add (int, int);
+        int Sub (int, int);
 };
-- 
2.34.1

